CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Welcome to the gematik contributing guide <!-- omit in toc -->
+ 
+Thank you for investing your time in contributing to our projects!
+ 
+<!-- do we have a Code of Conduct? What are the general rules of contribution? First issue and discussion?
+Read our [Code of Conduct](./CODE_OF_CONDUCT.md) to keep our community approachable and respectable. -->
```

Comment:
Dannn müssenn wir das auskommentieren, oder?

Reply:
Das war noch eine Frage, die ich hatte. Können da andere überhaupt mitwirken oder sind die FHIR-Geschichten komplett in unserer Hand und contributions nicht gewollt?

Reply:
Können wir nicht ausschließen, wollen wir eigentlich auch nicht. Z.b. das Repo https://github.com/gematik/fhir-profile-apovzd ist zwar private, aber ich habe es mit Christian Ruß von der NGDA geteilt, das er (theoretisch) mitwirken kann, da er Empfänger der Profile ist.

Reply:
Ist auskommentiert und scharf gestellt :)